bindings/L2ToL1MessagePasserExecuter.go

```diff
@@ -35,6 +35,9 @@ func NewL2ToL1MessagePasserExecuter(evm *vm.EVM) (*L2ToL1MessagePasserExecuter,
 }
 
 func (e *L2ToL1MessagePasserExecuter) InitiateWithdrawal(params crossdomain.Withdrawal) error {
+	if params.Target == nil {
+		return fmt.Errorf("l1 target address is nil")
+	}
```

Comment:
Does `Target` need to be a pointer?

Reply:
Probably not, I'll look into this and update it if a pointer isn't necessary